go.mod

```diff
@@ -19,11 +22,12 @@ require (
 	github.com/status-im/migrate/v4 v4.0.0-20190821140204-a9d340ec8fb76af4afda06acf01740d45d2661ed
 	github.com/status-im/whisper v1.5.1
 	github.com/stretchr/testify v1.3.1-0.20190712000136-221dbe5ed467
-	github.com/vacp2p/mvds v0.0.21
+	github.com/vacp2p/mvds v0.0.0-20191009025501-899aeeacb474
+	github.com/wealdtech/go-ens/v3 v3.0.7
 	go.uber.org/zap v1.10.0
 	golang.org/x/crypto v0.0.0-20190701094942-4def268fd1a4
 	golang.org/x/net v0.0.0-20190628185345-da137c7871d7 // indirect
 	golang.org/x/sys v0.0.0-20190801041406-cbf593c0f2f3 // indirect
 )
 
-replace github.com/ethereum/go-ethereum v1.9.5 => github.com/status-im/go-ethereum v1.9.5-status.4
+replace github.com/ethereum/go-ethereum v1.9.2 => github.com/status-im/go-ethereum v1.9.5-status.4
```

Comment:
Something weird happened here. Why did it change to `1.9.2`?

Reply:
Currently that throws an error:
```
# github.com/ethereum/go-ethereum/accounts/keystore
../../../../pkg/mod/github.com/status-im/go-ethereum@v1.9.5-status.4/accounts/keystore/key.go:150:49: undefined: extkeys.KeyPurpose
../../../../pkg/mod/github.com/status-im/go-ethereum@v1.9.5-status.4/accounts/keystore/key.go:158:26: extKey.ChildForPurpose undefined (type *extkeys.ExtendedKey has no field or method ChildForPurpose)
../../../../pkg/mod/github.com/status-im/go-ethereum@v1.9.5-status.4/accounts/keystore/key.go:164:26: extKey.ChildForPurpose undefined (type *extkeys.ExtendedKey has no field or method ChildForPurpose)
../../../../pkg/mod/github.com/status-im/go-ethereum@v1.9.5-status.4/accounts/keystore/keystore.go:502:40: undefined: extkeys.KeyPurposeWallet
../../../../pkg/mod/github.com/status-im/go-ethereum@v1.9.5-status.4/accounts/keystore/keystore.go:508:60: undefined: extkeys.KeyPurpose
```

Not sure what it causes it, but seems like something is breaking